api/woeip/apps/air_quality/serializers.py

```diff
@@ -122,6 +130,21 @@ def create(self, validated_data):
             ends_at=validated_data.get("ends_at"),
         )
 
+        # Create file hash, check if hash already exists.
+        dusktrak_upload_hash = str(hash(dustrak_upload_file.read()))
```

Comment:
At the risk of seeming pedantic `dusktrak` -> `dustrak`:

```suggestion
        dustrak_upload_hash = str(hash(dustrak_upload_file.read()))
```

The extra 'k' could cause confusion in defining variables